xml/System.Text.RegularExpressions/GroupCollection.xml

```diff
@@ -1351,9 +1278,11 @@
         <Parameter Name="value" Type="System.Object" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="item">This method is not supported.</param>
+        <summary>Implementation from <see cref="T:System.Collections.Generic.IList" />. This method is not supported.</summary>
+        <returns>This method is not supported.</returns>
```

Comment:
`<returns>This method is not supported.</returns>`

Isn't it better to say this method throws with NotSupportedException or something?

Reply:
I found two other places where returns tag says "This method is not supported."